src/app/services/person-adapter.service.ts

```diff
@@ -0,0 +1,110 @@
+import { Injectable } from '@angular/core';
+import { CreateExperiencesGQL, EditExperiencesDocument, Exact, Experience, InputMaybe, PersonWhere, PersonWithAllTypeFragment, PersonsWithAllGQL, PersonsWithAllQuery } from '../generated/graphql';
+import { Apollo } from 'apollo-angular';
+import { QLFilterBuilderService } from './ql-filter-builder.service';
+import { NzNotificationService } from 'ng-zorro-antd/notification';
+import { AbstractControl } from '@angular/forms';
+import { QueryRef } from 'apollo-angular';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class PersonAdapterService {
+  experiencesResponse: Experience[] | undefined = undefined;
+  personQueryRef: QueryRef<PersonsWithAllQuery, Exact<{ where?: InputMaybe<PersonWhere> | undefined; }>> | undefined = undefined;
+  editedPerson!: PersonWithAllTypeFragment | null | undefined;
+  isLoading: boolean = false;
+  
+  constructor(
+    private apollo: Apollo, private qlFilterService: QLFilterBuilderService, private notification: NzNotificationService, private ceGQL: CreateExperiencesGQL, private pGQL: PersonsWithAllGQL
```

Comment:
1. Lets format like this

```suggestion
    private apollo: Apollo, 
    private qlFilterService: QLFilterBuilderService, 
    private ceGQL: CreateExperiencesGQL, 
    private pGQL: PersonsWithAllGQL
```
2.  `NzNotificationService` -  for sure its not a part of this service.